packages/netlify-cms-widget-markdown/src/serializers/__tests__/index.spec.js

```diff
@@ -0,0 +1,41 @@
+import path from 'path';
+import fs from 'fs';
+import { markdownToSlate } from '../';
+
+describe('markdownToSlate', () => {
+  it('should handle use case from issue 3280', () => {
+    const mdast = fs.readFileSync(path.join(__dirname, '__fixtures__', 'issue_3280.md'));
```

Comment:
@erezrokah consider using in-place descriptions rather than codifying specific issues.